### ISSUE TITLE
The Voice

### ISSUE THREAD

**Issue by XanMag** [KVonGit/Quest-Group_Project#34]
Are we aiming for having an "author's voice" that is somewhat similar from transition to transition or are we relishing in the fact that this is a collaboration and we will celebrate the differences in writing styles (as will the players?)?  

On this note, it may be interesting to build in to the game a reason or explanation or setting - something - that transitions between narrators (writers)? Just a thought.  I don't think it's a good idea to have drastically different writing styles from "sub-world" to "sub-world" without some written acknowledgement that it is happening.  I also think it may be a worse idea to try and get all contributing authors to try and have the same "voice".

## **What do we think?**

**Comment by rheadkid** [comment]
...contemplating...

**Comment by rheadkid** [comment]
I think that we're not thinking about giving this any thought.

...but seriously, folks,

I believe we've decided to have the player 'inhabit' a new character during trips to dimensions. #37 

I would assume that this is not etched in stone. It would be pretty cool to have the player enter a dimension as (himself/herself) either before or after having inhabited / inhabiting) someone in said dimension.

Closing this for now, though.

---

If we want magic, I've got a lot of work to do.

...and I'd definitely like to have magic. (I think.)
